sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -13,13 +13,15 @@
 
 #include <CL/__spirv/spirv_types.hpp>         // for MatrixLayout, MatrixUse
 #include <sycl/access/access.hpp>             // for address_space, decorated
+#include <sycl/builtins.hpp>                  // for fabs
 #include <sycl/detail/defines_elementary.hpp> // for __SYCL_ALWAYS_INLINE
 #include <sycl/detail/pi.h>                   // for PI_ERROR_INVALID_DEVICE
 #include <sycl/exception.hpp>                 // for runtime_error
 #include <sycl/ext/oneapi/bfloat16.hpp>       // for bfloat16
-#include <sycl/group.hpp>                     // for group
-#include <sycl/multi_ptr.hpp>                 // for multi_ptr
-#include <sycl/sub_group.hpp>                 // for sub_group
+#include <sycl/ext/oneapi/experimental/annotated_ptr/annotated_ptr.hpp> // for annotated_ptr
```

Comment:
is the annotated_ptr header file addition needed?

Reply:
yes